app/Providers/AppServiceProvider.php

```diff
@@ -13,7 +13,7 @@ class AppServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        //
+        $this->app['view']->addNamespace('admin', base_path() . '/resources/views/backend/admin');
```

Comment:
it's not good because you need to add many view namespaces. I think you should remove it and no need to use view namespace. In controller just write this code
```php
return view('admin.admin.index');
```